src/table.scss

```diff
@@ -206,6 +206,7 @@ $block: #{$fd-namespace}-table;
       position: absolute;
       background-color: inherit;
       border-bottom: $fd-table-border;
+      margin-top: 0.05rem;
```

Comment:
I'm not sure what exactly this is fixing. 
the border around the cell is still missing:
<img width="1250" alt="Screen Shot 2021-04-28 at 12 03 33 PM" src="https://user-images.githubusercontent.com/39598672/116436542-7e820b80-a81a-11eb-82ca-8449fe10cfa7.png">

The first Header Column cell should have a border around all sides just like the second Header Column cell

Reply:
@InnaAtanasova that line of code actually fixes the bottom border for the cells itself. Previously they weren't visible due to browser's antialiasing or something like that. I'll add borders for the heading. Also, maybe it's a good idea to change darker right border to the same border color we use across table?

Reply:
@InnaAtanasova added missing borders. Please check.